tools/invocation_json_gen.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024, Data61, CSIRO (ABN 41 687 119 230)
```

Comment:
I don't think that's right. Even if it was based on a file by Data61/CSIRO, then the year should be the original year, not this year. But you probably changed enough that you can ignore the original copyright, this file is barely copyrightable as it is.

Reply:
I don't think I've added a new file to seL4 before, so I wasn't too sure what to do here.

Is 

# Copyright 2024, UNSW Sydney

appropriate?

Reply:
Depends, did you do this work as part of a contract for UNSW? Then yes, that's correct, though your contract may allow you to retain copyright and only require the work to be released as open source. If you're a student then you own the copyright and should use your name. See the [UNSW copyright site](https://subjectguides.library.unsw.edu.au/copyright/copyright_ownership) for more details.